torch/nn/modules/transformer.py

```diff
@@ -267,6 +267,19 @@ class TransformerEncoderLayer(Module):
     Neural Information Processing Systems, pages 6000-6010. Users may modify or implement
     in a different way during application.
 
+    forward() will use a special optimized implementation if all of the following
+    conditions are met:
+    - All conditions for the fast path of torch.nn.MultiheadAttention
```

Comment:
this is sloppy writing on my part that doesn't correspond to how it's actually enforced. will fix.